package.json

```diff
@@ -38,8 +38,8 @@
     "package:build:react-scripts": "cd $INIT_CWD && SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false react-scripts build",
     "package:build:types": "cd $INIT_CWD && tsc",
     "package:lint": "cd $INIT_CWD && eslint .",
-    "package:test": "cd $INIT_CWD && jest",
-    "package:test:withdb": "cd $INIT_CWD && yarn workspace @tupaia/database check-test-database-exists && DB_NAME=tupaia_test jest",
+    "package:test": "cd $INIT_CWD && jest --maxWorkers=50%",
```

Comment:
Set maxWorkers=50% here since it lead to a decent improvement in test performance in my local. These results seem pretty common: https://blog.developer.adobe.com/improve-jest-runner-performance-a8f56708ba94

Reply:
Weird, good catch. I get about a 15% speedup too on my M1 mac